src/benchq/problem_ingestion/hamiltonian_generation.py

```diff
@@ -15,6 +16,52 @@
 
 ### General generators
 
+def flatten_nx_graph(g):
+    new_ids = {}
+    count = 0
+    for node in g.nodes:
+        if node in new_ids:
+            pass
+        else:
+            new_ids[node] = count
+            count = count + 1
+    new_g = nx.relabel_nodes(g, new_ids)
+    return new_g
```

Comment:
Why not just use `convert_node_labels_to_integers` or `relabel_nodes`? Documentation can be found [here](https://networkx.org/documentation/stable/reference/relabel.html)

Reply:
Cool thanks